.travis.yml

```diff
@@ -1,18 +1,31 @@
 # Config file for automatic testing at travis-ci.org
 # This file will be regenerated if you run travis_pypi_setup.py
+sudo: false
+
+addons:
+  apt:
+    packages:
+      - libgdal-dev
 
 language: python
 python:
-  - 3.5
-  - 3.4
-  - 3.3
-  - 2.7
-  - 2.6
+  - "3.6"
+  - "3.5"
+  - "3.4"
 
 # command to install dependencies, e.g. pip install -r requirements.txt --use-mirrors
-install: pip install -U tox-travis
+install: pip install -U tox-travis numpy wheel
 
 # command to run tests, e.g. python setup.py test
 script: tox
 
+deploy:
+  provider: pypi
+  user: rasterfoundry
+  password:
+    secure: kjKWOTwTm2eaa10Pp4/ll480TfFxP5Lb+/9cuxC1gZNZ1JuuJ/y11RQTXCKKP5CxQV0JQ8zs1RAeg0ODXsX4f1edC714oWQx/2dl3BMb5rLo2Pbk5AkVQ50gN/P9d6cSVQrlv6U+CClr8w9FNWTZ/yIeL6bs2AbK9dUyTGx/i7GBOfkAcyW23kY4vYcemonwiTKtql1Pez9rPcwX3Q+3hyiU9rX0psK3H9vdH+VzrFR4+R9URtXoaYBrNDkO3IBy62qBLhMUOwrTDkV2IGjZ3O0w9sGmON3KieuwmbII4P40FRx5OLUnmSYXhQTkhXQ1OU4LnU62y4QYCw0B94tbM4gKJdLapOVO4SjqdCmVqBLV84A+DqdMxhYQYgtP8dUwDmmyOr4jS3AVpZY75Vgmd4V1tpMxaAJwBKAcXOueXExg4GaoT/1cSpZcPPojWcujNnJX2AnrTDHctJiKPWpkV9a0MQyQ3szgwgetl4YVbUoiyKjTPwQyoeTG+gCI5czwPZj35mqFeO+OCkJq3vvkIJ4K1f1sHcO541AJa24v8Akdp/VXPxQOG75/kK3EXoH/BfWkyJF4QH8GA+dksTQGozv9dtpTa7zpNh7FkJ8VFjxSvMcB3LjfzggvQ5N1CZNnWDIZ8JWbwMbNNCdn/mWEfMZ0/ChXt+U5BGdQUeqnCe0=
+  on:
+    tags: true
+    distributions: "sdist bdist_wheel"
```

Comment:
This needs to be at the same level as `provider` and `user`; it should not be under `on`.